mmv1/third_party/terraform/go.mod.erb

```diff
@@ -0,0 +1,32 @@
+<% autogen_exception -%>
+module github.com/hashicorp/terraform-provider-google<%= "-" + version unless version == 'ga' -%>
+
+require (
+    cloud.google.com/go/bigtable v1.10.1
```

Comment:
nit: Let's tab-space the file instead of spaces, that's what's used on both downstreams

Reply:
Sure! Will work on it. Thanks!